pkgs/development/tools/go-tools/default.nix

```diff
@@ -1,9 +1,9 @@
-{ buildGoModule
+{ buildGo122Module
```

Comment:
it is preferred way to override those in all-packages.nix

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults

Reply:
also, congratulations on first contribution !

Reply:
Thanks! Switched to the all-packages override method: https://github.com/NixOS/nixpkgs/pull/293629/commits/9b019e4dd465bb9f9e68371687fe66e4de9fe0bb.